### PR TITLE
Restore backup to new linode

### LINKED ISSUE #174: Implement restore backup to new Linode

### PR DESCRIPTION
**Pull request by SirCmpwn**
Closes #174 

Clicking restore now takes you here:

### CODE REVIEW

**Comment by eatonphil** [comment]
+1 to the select a datacenter text
- 1 to the back button there

**Comment by SirCmpwn** [comment]
Where do you want it? You told me in #420 that the cancel button on the right is a problem. Should I move it to the bottom? It looks weird there.

**Comment by eatonphil** [comment]
No, if it goes anywhere (I'm not really sure why/convinced it needs to be here) it should probably go near the "Add a Linode" text at the top.

**Comment by SirCmpwn** [comment]
It brings you back to the list of linodes to select a backup from.

**Comment by SirCmpwn** [comment]
Within the source selection.

File: src/linodes/create/components/BackupSelection.js [hunk 1]
```diff
@@ -25,12 +25,11 @@ export class BackupSelection extends Component {
...
-        <div className="pull-right">
-          <a href="#" className="back" onClick={goBack}>Back</a>
-        </div>
         <div key={l.id}>
-          <h3>{l.label}</h3>
-          <div className="backup-group">
+          <h3>
+            {l.label}
+          </h3>
+          <div className="backup-group" style={{ clear: 'both' }}>
```
Comment (eatonphil) [inline src/linodes/create/components/BackupSelection.js]: ```className="backup-group clearfix"```

Comment (SirCmpwn) [inline src/linodes/create/components/BackupSelection.js]: afaik we don't actually have a clearfix class

Comment (SirCmpwn) [inline src/linodes/create/components/BackupSelection.js]: Thanks, that's new.

Comment (SirCmpwn) [inline src/linodes/create/components/BackupSelection.js]: lol I'm actually using clearfix in this file a few lines up already

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**